src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReportTest.java

```diff
@@ -233,81 +238,75 @@ public void testConsiderCaseIfValidHearingStatusReportSummary2() {
-        ListingData reportListingData = report.processHearingsByHearingTypeRequest(listingDetails, submitEvents);
-
+        ListingData reportListingData = report.processHearingsByHearingTypeRequest(listingDetails, submitEvents,
+                DATE_FROM, DATE_TO);
         List<AdhocReportTypeItem> adhocReportTypeItemList = reportListingData.getLocalReportsDetail();
 
         AdhocReportType adhocReportType = adhocReportTypeItemList.get(0).getValue();
-        assertEquals("1", adhocReportType.getCaseReference());
+        assertEquals("3", adhocReportType.getCaseReference());
         assertEquals("Y", adhocReportType.getLeadCase());
         assertEquals("471", adhocReportType.getHearingDuration());
         assertEquals("multiRef, subMulti", adhocReportType.getMultSub());
-        assertEquals("Hearing", adhocReportType.getHearingType());
+        assertEquals("Costs Hearing", adhocReportType.getHearingType());
         assertEquals("", adhocReportType.getHearingTelConf());
-        assertEquals("Y", adhocReportType.getJudicialMediation());
         assertEquals("clerk1", adhocReportType.getHearingClerk());
-
-        assertEquals("1970-06-01T00:00:00.000", adhocReportType.getDate());
-
-
+        assertEquals("2021-06-01 00:00:00.000", adhocReportType.getDate());
     }
 
     @Test
-    public void testConsiderCaseIfNullMultSubInReportDetail() {
-
-
+    public void testConsiderCaseIfNullMultiSubInReportDetail() {
         ListingDetails listingDetails = new ListingDetails();
         listingDetails.setCaseTypeId(NEWCASTLE_LISTING_CASE_TYPE_ID);
         ListingData listingData = new ListingData();
         listingDetails.setCaseData(listingData);
         HearingsByHearingTypeReport report = new HearingsByHearingTypeReport();
-        DateListedTypeItem dateListedTypeItem = createHearingDateListed("1970-06-01T00:00:00.000",
+        DateListedTypeItem dateListedTypeItem = createHearingDateListed("2021-06-01T00:00:00.000",
                 HEARING_STATUS_HEARD);
         List<HearingTypeItem> hearings = createHearingCollection(createHearing(HEARING_TYPE_JUDICIAL_HEARING, "JM",
                 dateListedTypeItem));
         SubmitEvent submitEvent = new SubmitEvent();
         CaseData caseData = new CaseData();
         caseData.setHearingCollection(hearings);
         submitEvent.setCaseData(caseData);
-        ListingData reportListingData = report.processHearingsByHearingTypeRequest(listingDetails, List.of(submitEvent));
+        ListingData reportListingData = report.processHearingsByHearingTypeRequest(listingDetails, List.of(submitEvent),
+                DATE_FROM, DATE_TO);
         List<AdhocReportTypeItem> adhocReportTypeItemList = reportListingData.getLocalReportsDetail();
         AdhocReportType adhocReportType = adhocReportTypeItemList.get(0).getValue();
         assertEquals("0 -  Not Allocated, 0 -  Not Allocated", adhocReportType.getMultSub());
     }
 
     @Test
```

Comment:
May be a test where a hearing collection has two hearings one in the range and one out of range?

Reply:
New test created to test this scenario